test/instances/eeth/EETHHyperdrive.t.sol

```diff
@@ -0,0 +1,618 @@
+import { EETHTarget4Deployer } from "contracts/src/deployers/eeth/EETHTarget4Deployer.sol";
+import { HyperdriveFactory } from "contracts/src/factory/HyperdriveFactory.sol";
+import { EETHConversions } from "contracts/src/instances/eeth/EETHConversions.sol";
+import { IERC20 } from "contracts/src/interfaces/IERC20.sol";
+import { IHyperdrive } from "contracts/src/interfaces/IHyperdrive.sol";
+import { IEETHHyperdrive } from "contracts/src/interfaces/IEETHHyperdrive.sol";
+import { AssetId } from "contracts/src/libraries/AssetId.sol";
+import { ETH } from "contracts/src/libraries/Constants.sol";
+import { FixedPointMath, ONE } from "contracts/src/libraries/FixedPointMath.sol";
+import { HyperdriveMath } from "contracts/src/libraries/HyperdriveMath.sol";
+import { ERC20ForwarderFactory } from "contracts/src/token/ERC20ForwarderFactory.sol";
+import { ERC20Mintable } from "contracts/test/ERC20Mintable.sol";
+import { InstanceTest } from "test/utils/InstanceTest.sol";
+import { HyperdriveUtils } from "test/utils/HyperdriveUtils.sol";
+import { Lib } from "test/utils/Lib.sol";
+
+contract EETHHyperdriveTest is InstanceTest {
+    using FixedPointMath for uint256;
+    using HyperdriveUtils for IHyperdrive;
+    using Lib for *;
+    using stdStorage for StdStorage;
+
+    // The mainnet liquidity pool.
+    ILiquidityPool internal constant POOL =
+        ILiquidityPool(0x308861A430be4cce5502d0A12724771Fc6DaF216);
+
+    // The mainnet EETH token.
+    IeETH internal constant EETH =
+        IeETH(0x35fA164735182de50811E8e2E824cFb9B6118ac2);
+
+    address internal constant MEMBERSHIP_MANAGER =
+        0x3d320286E014C3e1ce99Af6d6B00f0C1D63E3000;
+
+    // Whale accounts.
+    address internal EETH_TOKEN_WHALE =
+        address(0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee);
+    address[] internal EETHTokenWhaleAccounts = [EETH_TOKEN_WHALE];
+
+    // The configuration for the Instance testing suite.
+    InstanceTestConfig internal __testConfig =
+        InstanceTestConfig({
+            name: "Hyperdrive",
+            kind: "EETHHyperdrive",
+            baseTokenWhaleAccounts: new address[](0),
+            vaultSharesTokenWhaleAccounts: EETHTokenWhaleAccounts,
+            baseToken: IERC20(ETH),
+            vaultSharesToken: IERC20(address(EETH)),
+            shareTolerance: 1e5,
+            minTransactionAmount: 1e15,
+            positionDuration: POSITION_DURATION,
+            enableBaseDeposits: true,
+            enableShareDeposits: true,
+            enableBaseWithdraws: false,
+            enableShareWithdraws: true
+        });
+
+    /// @dev Instantiates the Instance testing suite with the configuration.
+    constructor() InstanceTest(__testConfig) {}
+
+    /// @dev Forge function that is invoked to setup the testing environment.
+    function setUp() public override __mainnet_fork(20_362_343) {
+        // Invoke the Instance testing suite setup.
```

Comment:
Nit:

```suggestion
        // Invoke the instance testing suite setup.
```

Reply:
I think this is from codegen. I'll update the template too so it doesn't keep happening